auth/src/main/java/org/aerogear/mobile/auth/user/UserPrincipalImpl.java

```diff
@@ -157,21 +157,24 @@ public static Builder newUser() {
 
     @Override
     public String toString() {
-        String roleNames = "";
-        Iterator<UserRole> i = roles.iterator();
-        if (i.hasNext()) {
-            //first element
-            roleNames.concat("[").concat(i.next().getName());
-            while(i.hasNext()) {
-                roleNames.concat(", ").concat(i.next().getName());
+        final StringBuilder sb = new StringBuilder();
+        sb.append("[");
```

Comment:
when there are no roles... we want to render to following -> [], right ?

Reply:
Yes that's correct